sim/dex-data.ts

```diff
@@ -69,14 +72,13 @@ export class BasicEffect {
 	/**
 	 * Effect type.
 	 */
-	effectType: EffectTypes;
+	effectType: EffectTypes; // TODO: EffectType
```

Comment:
I think `EffectTypes` should be renamed `EffectType`.

Reply:
Agreed.